.azure-pipelines/ci-build.yml

```diff
@@ -616,11 +616,33 @@ extends:
                 displayName: "Get Kiota's version-number from .csproj"
               - pwsh: $(Build.SourcesDirectory)/scripts/update-vscode-releases.ps1 -version $(artifactVersion)$(versionSuffix) -packageJsonFilePath $(Build.SourcesDirectory)/vscode/microsoft-kiota/package.json -runtimeFilePath $(Build.SourcesDirectory)/vscode/npm-package/runtime.json -binaryFolderPath $(Build.ArtifactStagingDirectory)/Binaries
                 displayName: "Update VSCode extension version-number"
-              - script: npm i -g @vscode/vsce
-                displayName: "Install vsce"
-              - script: npm ci
-                displayName: "Install dependencies"
-                workingDirectory: $(Build.SourcesDirectory)/vscode/microsoft-kiota
+             
+              - script: |
+                  npm i -g @vscode/vsce rimraf
+                displayName: "Install Global dependencies"
+                workingDirectory: $(Build.SourcesDirectory)/vscode
+
+              - script: |
+                  cd npm-package
```

Comment:
you don't need to cd if you change the working directory below (multiple times)

Reply:
I see what you mean